llvm/lib/IR/Verifier.cpp

```diff
@@ -3765,9 +3765,28 @@ void Verifier::visitCallBase(CallBase &Call) {
           "Return type cannot be x86_amx for indirect call!");
   }
 
-  if (Function *F = Call.getCalledFunction())
+  if (Function *F = Call.getCalledFunction()) {
     if (Intrinsic::ID ID = (Intrinsic::ID)F->getIntrinsicID())
       visitIntrinsicCall(ID, Call);
+    // TODO: Move this to target dependent IR verifier once we have it.
```

Comment:
Remove this comment. This belongs in the generic verifier. The calling conventions, while intended to only be used by certain targets, are not target code. The calling conventions are global invariants